components/selectors/parser.rs

```diff
@@ -1732,19 +1746,44 @@ pub mod tests {
         }
     }
 
-    fn parse<'i>(input: &'i str) -> Result<SelectorList<DummySelectorImpl>,
-                                           ParseError<'i, SelectorParseError<'i, ()>>> {
+    fn parse<'i>(input: &'i str)
+                 -> Result<SelectorList<DummySelectorImpl>, ParseError<'i, SelectorParseError<'i, ()>>>
```

Comment:
No need to change this now, but next time you’d like to make whitespace-only changes please make them in a separate commit.